use/rootless.md

```diff
@@ -1,30 +1,40 @@
 ---
 title: "Running Rootless"
 description: Rootless containers and rocker
+execute:
+  eval: false
```

Comment:
Is this necessary?
My understanding is that if the file extension is `md`, this is unnecessary because Quarto CLI determines that there is no executable code block.

Reply:
Oops! Removed